eng/pipelines/templates/steps/generate-nightly-branch-name.yml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - pwsh: |
+      $branchName = $env:NIGHTLYBRANCHOVERRIDE
+      if (!$branchName) { 
+        $formattedDate = Get-Date -Format 'yyyy-MM-dd'
+        $branchName = "nightly/$formattedDate"
```

Comment:
Will this actually be ran nightly or just daily or multiple times a day? If this branch is going to be longer lived we should also try to follow our standard branch naming guidelines.

Reply:
What about `release/daily-$formattedDate`